buildenv/docker/jdk9/armhf_CC/arm-linux-gnueabihf/Dockerfile

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2017, 2017 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
```

Comment:
The instructions to run were completely removed. I'm going to merge anyway and they can be added afterwards.

Reply:
Thanks for the merge.
I feel quite strongly about the use of volumes when using Docker for iterative development (it's less of an issue for automated build), but my realisation was that I was about to go to war over a comment. The Dockerfile has no dependency one way or another, so any comment about how to use it within the file is little more than a suggestion (but with the potential to confuse). Since the usage can vary depending on context I would recommend not putting that information inside the Dockerfile, but keeping it in the external documentation, which can be specialised accordingly.